docker/CMSRucioClient/scripts/CMSRSE.py

```diff
@@ -461,6 +472,7 @@ def _set_protocols(self):
         except (RSEProtocolNotSupported, RSENotFound):
             current_protocols = []
 
+        new_changes = False
```

Comment:
This is to track if a change in protocol took place, earlier we were just returning None. This was not an issue, but the logs were misleading.